packages/keypad-context/src/keypad-context.tsx

```diff
@@ -32,7 +32,7 @@ export function StatefulKeypadContextProvider(props: Props) {
     // whether or not to display the keypad
     const [keypadActive, setKeypadActive] = useState<boolean>(false);
     // used to communicate between the keypad and the Renderer
-    const [keypadElement, setKeypadElement] = useState<KeypadAPI | null>();
+    const [keypadElement, setKeypadElement] = useState<any>();
```

Comment:
😬  `any` ?

Reply:
Yeah. I tried to move types from `@khanacademy/math-input` and things quickly spiraled. I think short-term it's good to have this separate package, but long-term we need to find a better solution.